install/install.sh

```diff
@@ -41,6 +41,9 @@ do
                     exit 1
                 fi
             fi
+            if [  -f "$file" ]; then
+                break
+            fi
```

Comment:
Why this?

Reply:
You obviously didn't follow the Slack discussion; it is because the installer keeps looping otherwise.

Reply:
I had an error while installing it that the terminal entered the install mysql condition though it was already installed and ended up forming an infinite loop just printing on the screen "* Installing MySQL (root password will be empty!) " so I added a condition to break the loop from inside if the file is present in case

Reply:
Thanks for the update :D